docs/introduction/welcome.stories.mdx

```diff
@@ -29,7 +29,7 @@ import {
 
 Set up a new app with Circuit UI or add it to an existing project.
 
-[Get started](Introduction/Getting-started)
+[Get started](Introduction/Getting-Started/Page)
```

Comment:
There were several broken links on this page

Reply:
Thanks for fixing them! 🙌